res/css/_common.scss

```diff
@@ -110,6 +110,10 @@ textarea {
     color: $primary-fg-color;
 }
 
+.mx_HiddenFocusable {
```

Comment:
Maybe add a comment to explain that normally the focus outline is helpful for a11y, but this can be used to annotate the places where that isn't desired.